locals.tf

```diff
@@ -3,4 +3,20 @@ locals {
     scope      = "terraform-managed"
     repository = "jenkins-infra/azure-net"
   }
+
+  vpn = {
+    shorthostname = "vpn-test"
+    username      = "jenkins-infra-team"
+    allowed_ips = {
```

Comment:
What’s this for?

Reply:
I'm recreating the VPN as code, these IPs are the current ones allowed in the Network Security Group applied to the VPN subnet.

<img width="2110" alt="image" src="https://user-images.githubusercontent.com/91831478/204912390-b89674ae-36a0-4236-a4e4-f8995ebe357f.png">

Reply:
I realized I haven't updated the PR body with refs, let me add them.

Reply:
that's confusing naming.

from the code locals you wrote it seems like it's allowed_ips for the VPN.

but it's actually a security rule for port 22

I would change it to at least:
ssh_allowed_ips

Reply:
You're totally right, let me open a PR to fix that.